config/application.properties

```diff
@@ -121,19 +121,20 @@ spring.datasource.removeAbandoned = true
 #spring.jpa.generate-ddl = true
 #spring.jpa.hibernate.ddl-auto = create-drop
 #spring.jpa.show-sql = true
-spring.jpa.properties.hibername.show_sql = true
```

Comment:
Notice I had misspelled "hibernate" here.  These properties would have worked, but I was accidentally not using them (and the ones below under "logging.level.org.hibernate" were controlling this.